flybywiresim/a32nx
# fix(displays): remove extra backlight bleed from ND and PFD

Fixes #6425

## Summary of Changes

Removed extra backlight bleed div from index.tsx of PFD and ND, as this is already supplied in displayUnit.tsx.

There should be no visible change for end users as result of this PR. 

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): 2Cas#1022

## Testing instructions

1. Load in C&D at any airfield, or in mid air and power on, time should be at night.
2. Turn up brightness on PFD/ND
3. Observe that the LCD backlight effect is still present on both the PFD and ND


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
